core/edit/edit.talon

```diff
@@ -185,3 +185,5 @@ redo that: edit.redo()
 # Save
 file save: edit.save()
 file save all: edit.save_all()
+
+go mid: user.go_mid()
```

Comment:
There is "go top" and "go bottom" which are file commands, so "go mid" could be interpreted as file or line in. The existing line commands are "go line start | head" and "go line start | tail", so I would be tempted to make this "go line mid"? Suppose you could do "go line mid | mid"? However, my only hesitation with using "mid" on its own is that there is https://github.com/talonhub/community/blob/5ebcea7e662f86eb076ac595fc7d58e242fb6822/plugin/mouse/mouse.talon#L23, "mid click"... I've used "click" as an alternative for "touch" before, so this would then conflict with "mid click".

Reply:
What do you think about making "go" optional? `[go] line mid`?

Reply:
Ya I think that'd be fine

Reply:
updated - thanks for the feedback!